internal/tools/go.mod

```diff
@@ -4,16 +4,16 @@ go 1.17
 
 require (
 	github.com/client9/misspell v0.3.4
-	github.com/golangci/golangci-lint v1.45.2
+	github.com/golangci/golangci-lint v1.45.0
```

Comment:
Is this downgrade to deal with the modified tag in one of its dependencies?  Can that dependency version be excluded instead?

Reply:
Not sure what happened, was definitely not intentional.